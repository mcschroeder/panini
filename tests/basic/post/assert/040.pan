assert : { b:𝔹 | b = true } → 𝟙

f40 : {x:bool|x=true} -> {v:unit|?}
f40 = \x:bool. assert x
