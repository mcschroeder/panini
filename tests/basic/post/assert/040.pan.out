f40 : {x:𝔹 | x = true} → {v:𝟙 | true}